src/components/tree/phyloTree/layouts.ts

```diff
@@ -389,7 +395,8 @@ export const mapToScreen = function mapToScreen(this: PhyloTreeType): void {
       spanX = minimumXAxisSpan;
     }
     /* In rectangular mode, if the tree has been zoomed, leave some room to display the (clade's) root branch */
-    if (this.layout==="rect" && this.zoomNode.n.arrayIdx!==0) {
+    // FIXME: is this right?
+    if (this.layout==="rect" && this.zoomNode.n.arrayIdx!==0 || getBranchVisibility(this.zoomNode) === "hidden") {
```

Comment:
What's the intention of this change? Is the change meant for non-rectangular trees only (if so, I'd add a `this.layout!=='rect'` conditional)? Or were you intending

```js
    if (this.layout==="rect" && (this.zoomNode.n.arrayIdx!==0 || getBranchVisibility(this.zoomNode) === "hidden"))
```
which would add LHS padding when the root was hidden. 

P.S. This entire aspect of Auspice is very ad-hoc and could use broader improvements.

Reply:
It's meant for rectangular trees only, so yes to your code snippet. I forgot to test other layouts, will take a look to see if it's easy.

Reply:
Updated the condition to your code snippet:

https://github.com/nextstrain/auspice/blob/29b03a9a193cd80ace27b3373490594f56eacb89/src/components/tree/phyloTree/layouts.ts#L402